.github/workflows/ci-cd.yml

```
@@ -136,4 +136,4 @@ jobs:
           version_label: "github-action--${{ steps.deploy-time.outputs.formattedTime }}"
           region: ap-northeast-2
           deployment_package: deploy.zip
-          wait_for_environment_recovery: 20
\ No newline at end of file
+          wait_for_environment_recovery: 100
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)